tmt/utils/templates.py

```diff
@@ -283,6 +284,21 @@ def _template_filter_listed(  # type: ignore[reportUnusedFunction,unused-ignore]
         join=join))
 
 
+def _template_filter_web_git_url(  # type: ignore[reportUnusedFunction,unused-ignore]
+        path_str: str,
+        url: str,
+        ref: str) -> str:
+    """
+    Sanitize git url using :py:meth:`tmt.utils.web_git_url`
+
+    .. code-block:: jinja
+
+        {{ "/"|web_git_url(STORY.fmf_id.url, STORY.fmf_id.ref) }}
```

Comment:
Could you maybe add a concrete example of the output as we do for other `_template_filter_*` methods?

Reply:
What about something like this: d13023cf ?